lib/commands/context.js

```diff
@@ -117,12 +116,21 @@ commands.mobileGetContexts = async function mobileGetContexts () {
   return await webviewHelpers.getWebViewsMapping(this.adb, opts);
 };
 
-helpers.switchContext = async function switchContext (name) {
+helpers.getContextsWithWebviewsMapping = function getContexts (webviewsMapping) {
```

Comment:
Please align the variable and function names

Reply:
this function has a side effect by changing the `contexts` property. Could we change its name to reflect that, for example: `assignContexts`

Reply:
Addressed as https://github.com/appium/appium-android-driver/pull/699/commits/d2f7f406934ee63509c5f604ace0c12bdcc11541